### ISSUE TITLE
Along returns incorrect coordinates

### ISSUE THREAD

**Issue by dimon82** [Turfjs/turf#1763]
var line = turf.lineString([[-82.461, 35.3187], [144.985, -37.813]]);
    var segment = turf.along(line, 1500);
    console.log(segment.geometry.coordinates);

this outputs `262.6856469517484, 30.179864583770968`

Because of this output when I do the LineString in MapBox between `[-82.461, 35.3187]` and `262.6856469517484, 30.179864583770968`, it looks like this:
![image](https://user-images.githubusercontent.com/5393145/66091398-75e12a80-e53c-11e9-82f1-ce0d80a292b7.png)

Isn't it supposed to output `-97.314353, 30.179864583770968` instead?

**Comment by matths** [comment]
@dimon82 did you ever get some hint why this is? What was your solution?

**Comment by smallsaucepan** [comment]
You're correct @dimon82. along should be normalising the longitude to between -180 and 180 before returning. Thanks for reporting (although I know it's been a while).
